### PR TITLE
Modal tests

### PR DESCRIPTION
**Pull request by rishichawda**

### CODE REVIEW

File: src/modal/index.js [hunk 1]
```diff
@@ -22,60 +22,53 @@ class Modal extends Component {
...
-              { footer }
-            </div>
-          )
-          : null
-          }
-        </div>
-      ) : children;
+    const classes = cx(
+      theme.modal,
+      open === true ? theme['d-block'] : theme['d-none'],
```
Comment (prasanna1211) [inline src/modal/index.js]: shouldn't this be just `open ? theme .... : ....`

File: src/modal/index.js [hunk 2]
```diff
@@ -22,60 +22,53 @@ class Modal extends Component {
...
+      className,
+    );
+    const backdrop = cx(
+      theme['modal-backdrop'],
+      open === true ? theme['d-block'] : theme['d-none'],
```
Comment (prasanna1211) [inline src/modal/index.js]: same unnecessary true condition

File: src/modal/index.js [hunk 3]
```diff
@@ -22,60 +22,53 @@ class Modal extends Component {
...
-           </div>
-          }
-          <div className={theme.close} onClick={this.closeModal} />
-        </div>
+      <div
+        className={backdrop}
+        onClick={closeOnBackdropClick ? this.closeModal : undefined}
+      >
+        {open && (
+          <div id="modal" className={classes}>
```
Comment (prasanna1211) [inline src/modal/index.js:79]: Readability looks to be little less. Can you split this into sub render functions ?

**Review by prasanna1211 — CHANGES_REQUESTED** [review]
Made some comments

File: src/modal/index.js (hunk 3 above)
Comment (rishichawda) [reply src/modal/index.js:79]: Sure. I'll make changes and push.

**Update**: Refactored title and footer render to separate functions.

**Comment by prasanna1211** [comment]
LGTM

**Pull request merged**